.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!---
+  if some of the following sections doesn't apply,
+  delete the section.
+
+  Also feel free to delete the comments.
+--->
+
+## Depends
+
+<!--- All PR or issues that are required to be closed / merged before this can be merged --->
+- [ ] #100
+
+## Problem
+
+<!--- Restate the problem addressed by the PR here --->
+
+## Solution
+
+<!--- Restate the basic ideas behind your solution --->
+<!--- Here it is also a good space to put details of your implementation --->
+
+## Related
```

Comment:
What do you mean by related ?
If you are meaning issues that will be closed by this PR, I prefer to make `Fix #99` commit message mandatory and use `linked issues` feature which seams more reliable.

Reply:
I think @EduardoRFS finds it useful to have related PRs and issues that dont necessarily haven't to be closed when current PR gets merged. I agree with him. Mostly because merge commits will then contain very useful metadata.

Reply:
@oteku anything related to the current PR, they may be issues non closed that we're slowly addressing, or that we don't depend on, just as a way to track how this PR relates to others.